tests/integration/test_worker.py

```diff
@@ -215,3 +215,13 @@ def test_sigusr1(self):
             worker.expect('Consumer started')
             worker.send_signal(signal.SIGUSR1)
             worker.expect('traceback.format_stack')
+
+    def test_batch(self):
+        """Batch tasks are run"""
+        li = [tasks.echo.subtask(args=("foo %i" % i, )) for i in range(2)]
```

Comment:
While I see the point of having `args` and `kwargs` explicitly, it doesn't look as nice as just:

```python
li = [tasks.echo.subtask("foo %i" % i) for i in range(2)]
```

Celery went with this approach, but I am not sure how to fit the `host` argument in this case.